src/helper/base64.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Base64 encoding/decoding (RFC1341)
+ * Copyright (c) 2005-2011, Jouni Malinen <j@w1.fi>
+ *
+ * This software may be distributed under the terms of the BSD license.
+ * See README for more details.
```

Comment:
If you plan to upstream this in OpenOCD, please review the license text.
I think BSD license is ok, but where is the README ?
Maybe it would be enough to keep the copyright and replace the license text with a simple SPDX tag.
For an alternative, I have quickly searched for an implementation of base64 enc/dec under GPLv2-or-later, but nothing so far.